azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/create/CreateDatabaseDao.java

```diff
@@ -0,0 +1,53 @@
+package bio.terra.workspace.azureDatabaseUtils.create;
+
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class CreateDatabaseDao {
+  private final NamedParameterJdbcTemplate jdbcTemplate;
+
+  @Autowired
+  public CreateDatabaseDao(NamedParameterJdbcTemplate jdbcTemplate) {
+    this.jdbcTemplate = jdbcTemplate;
+  }
+
+  public void createDatabase(String databaseName) {
+    // databaseName should already be validated by the service layer
+    // CREATE DATABASE does not like bind parameters
+    jdbcTemplate.update("CREATE DATABASE " + databaseName, Map.of());
+  }
+
+  public String createRole(String roleName, String userOID) {
```

Comment:
It would be good to add JavaDoc. Specifically, I wonder what "OID" stands for and if there are assumptions about the format, and I wonder what the return value from this function signifies.